cuebot/src/main/java/com/imageworks/spcue/dao/postgres/NestedWhiteboardDaoJdbc.java

```diff
@@ -290,12 +290,15 @@ private static final NestedJob mapResultSetToJob(ResultSet rs) throws SQLExcepti
             "host_stat.ts_ping,"+
             "host.int_cores,"+
             "host.int_cores_idle,"+
+            "host.int_gpu," +
+            "host.int_gpu_idle," +
             "host.int_mem,"+
             "host.int_mem_idle,"+
             "host.str_lock_state,"+
             "host.str_tags,"+
             "host.b_comment,"+
             "host.int_thread_mode,"+
+            "host_stat.str_os,"+
```

Comment:
nit: we should standardize whitespace, at least for new lines (i.e. add space between string and `+` in the current version).

Reply:
Good point. I went ahead and standardized the whitespace before and after operators for this file. I also added a whitespace after each `,` in the sql queries to help with debugging readability. PTAL.